cicecore/cicedynB/analysis/ice_history_shared.F90

```diff
@@ -152,32 +152,52 @@ module ice_history_shared
          avgct(max_nstrm)   ! average sample counter
 
       logical (kind=log_kind), public :: &
-         igrd (nvar), &        ! true if grid field is written to output file
-         igrdz(nvarz)          ! true if category/vertical grid field is written
+         igrd (nvar_grd), &    ! true if grid field is written to output file
+         igrdz(nvar_grdz)      ! true if category/vertical grid field is written
 
       character (len=25), public, parameter :: &
          tcstr = 'area: tarea'          , & ! vcellmeas for T cell quantities
          ucstr = 'area: uarea'          , & ! vcellmeas for U cell quantities
+         ncstr = 'area: narea'          , & ! vcellmeas for N cell quantities
+         ecstr = 'area: earea'          , & ! vcellmeas for E cell quantities
          tstr2D  = 'TLON TLAT time'     , & ! vcoord for T cell quantities, 2D
          ustr2D  = 'ULON ULAT time'     , & ! vcoord for U cell quantities, 2D
+         nstr2D  = 'NLON NLAT time'     , & ! vcoord for N cell quantities, 2D
+         estr2D  = 'ELON ELAT time'     , & ! vcoord for E cell quantities, 2D
          tstr3Dz = 'TLON TLAT VGRDi time',& ! vcoord for T cell quantities, 3D
          ustr3Dz = 'ULON ULAT VGRDi time',& ! vcoord for U cell quantities, 3D
+         nstr3Dz = 'NLON NLAT VGRDi time',& ! vcoord for N cell quantities, 3D
```

Comment:
I don't think we will need some of these options on the N and E grids, but it's okay to have them available for future use.  The B- vs C-grid choice should only impact the dynamics variables, which do not have vertical, ITD (ncat) or FSD dimensions.

Reply:
These 4d options already exists for the ULON ULAT, so that's why I added them for the N and E grids.  It could be that we can get rid of some of them at some point.

Reply:
I've added a tick box to the project page.